modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -1835,6 +1835,17 @@ protected void updatePropertyForArray(CodegenProperty property, CodegenProperty
             property.complexType = innerProperty.baseType;
         } else {
             property.isPrimitiveType = true;
+			property.isBoolean = innerProperty.isBoolean;
```

Comment:
Alignment

Reply:
@marcindanek85 can you fix the alignment using spaces instead of tabs when you've time?

Let us know if you need help.

Reply:
I've fixed this in this PR: https://github.com/swagger-api/swagger-codegen/pull/7265